pkgs/spark-wallet/supplement.nix

```diff
@@ -1,4 +1,4 @@
-# This file has been generated by node2nix 1.6.1. Do not edit!
+# This file has been generated by node2nix 1.6.0. Do not edit!
```

Comment:
Just out of curiosity why did we downgrades node2nix versions?

Reply:
1.6.0 is the latest version in stable. The last generation was done by clefru - not sure where he got 1.6.1 from.